packages/dcos-checks/extra/dcos-checks.timer

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=DC/OS Checks Timer: timer for DC/OS Checks service
+[Timer]
+Unit=dcos-checks.service
+OnBootSec=15min
```

Comment:
Why wait 15 minutes after boot before we run checks? It should be OK if checks run and fail while other components are still starting up.